### PR TITLE
Adding run_started_at to query comment

### PR DESCRIPTION
**Pull request by bisset-a**
Adding `run_started_at` to the query comment

### CODE REVIEW

File: macros/query_comment.sql [hunk 1]
```diff
@@ -23,6 +23,7 @@
             node_resource_type=node.resource_type,
             node_meta=node.config.meta,
             node_tags=node.tags,
+            run_started_at=run_started_at.strftime("%Y-%m-%d %H:%M:%S"),
```
Comment (NiallRees) [inline macros/query_comment.sql:26]: Wondering if we can go ISO 8601 UTC here?

**Comment by NiallRees** [comment]
Thanks for contributing this @bisset-a 🙌

File: macros/query_comment.sql (hunk 1 above)
Comment (bisset-a) [reply macros/query_comment.sql]: Hey @NiallRees!! I've just updated and tested:
image

**Comment by bisset-a** [comment]
> Thanks for contributing this @bisset-a 🙌

No problems @NiallRees , thanks for providing the macros 😄 they'll be very useful for us!

File: macros/query_comment.sql [hunk 2]
```diff
@@ -23,6 +23,7 @@
             node_resource_type=node.resource_type,
             node_meta=node.config.meta,
             node_tags=node.tags,
+            run_started_at=run_started_at.astimezone(modules.pytz.utc).isoformat(),
             full_refresh=flags.FULL_REFRESH,
```
Comment (robscriva) [inline macros/query_comment.sql:27]: Should this go up higher with the `invocation_id` and other non-node specific details?

**Review by NiallRees — APPROVED** [review]

**Pull request merged**